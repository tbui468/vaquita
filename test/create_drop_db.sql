create database sol;
create database orion;
create database sirius;
drop database sol;
drop database orion;
drop database sirius;
exit;
