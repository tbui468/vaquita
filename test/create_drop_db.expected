created database sol
created database orion
created database sirius
dropped database sol
dropped database orion
dropped database sirius
